app/views/items/index.html.erb

```diff
@@ -1,6 +1,6 @@
 <div>
 	<% @items.each do |item| %>
-	<div class="item_card">
+	<div class="item_card index_item_<%= item.id %>">
```

Comment:
I like the naming conventions for css classes so far.

Reply:
This is pretty cool. Glad we figured this out.